services/common/src/models/documents/document.ts

```diff
@@ -163,10 +163,10 @@ export class MineDocument implements IMineDocument {
     _userRoles: string[] = [],
     is_latest_version: boolean = this.is_latest_version
   ) {
-    const canModify = is_latest_version && !this.is_archived;
-    if (!this.mine_document_guid) return [];
+    const canModify = is_latest_version && !this.is_archived && this.mine_document_guid;
+    const canView = this.file_type === ".pdf" && this.document_manager_guid;
```

Comment:
For cypress tests. So that the actions column can be there before we save the file (download/view actions require document_manager_guid, not mine_document_guid, but I kept the other actions from being allowed)